tests/pipelines/test_pipelines_common.py

```diff
@@ -607,3 +617,125 @@ def add(number, extra=0):
 
         outputs = [item for item in dataset]
         self.assertEqual(outputs, [[{"id": 2}, {"id": 3}, {"id": 4}, {"id": 5}]])
+
+    def check_models_equal_pt(self, model1, model2):
+        models_are_equal = True
+        for model1_p, model2_p in zip(model1.parameters(), model2.parameters()):
+            if model1_p.data.ne(model2_p.data).sum() > 0:
+                models_are_equal = False
+
+        return models_are_equal
+
+    def check_models_equal_tf(self, model1, model2):
+        models_are_equal = True
+        for model1_p, model2_p in zip(model1.weights, model2.weights):
+            if np.abs(model1_p.numpy() - model2_p.numpy()).sum() > 1e-5:
+                models_are_equal = False
+
+        return models_are_equal
+
+    @slow
+    @require_torch
+    def test_load_default_pipelines_pt(self):
```

Comment:
I test here that the default model that is loaded by calling `pipeline(<task_name>)` indeed loads the corresponding model. Weights are compared to be sure it's actually exactly the same model.

Reply:
This test is run for all pipelines and should in general serve as a good test to make sure all default pipelines work as expected